data/presets/man_made/charge_point.json

```diff
@@ -0,0 +1,33 @@
+{
+    "icon": "fas-charging-station",
```

Comment:
Reusing the same icon is not great to distinguish the features. 
I'm not sure what would be the best solution here, but maybe this preset could just use the `fas-plug` icon?